skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerMetrics.scala

```diff
@@ -115,50 +118,226 @@ object ConsumerMetrics {
-            a <- age.foldMapM { age =>
-              ageSummary
-                .labels(clientId, topic)
-                .observe(age.toNanos.nanosToSeconds)
-            }
-          } yield a
+    override def poll(topic: Topic, bytes: Int, records: Int, age: Option[FiniteDuration]): F[Unit] = {
+      for {
+        _ <- recordsSummary
+          .labels(clientId, topic)
+          .observe(records.toDouble)
+        _ <- bytesSummary
+          .labels(clientId, topic)
+          .observe(bytes.toDouble)
+        a <- age.foldMapM { age =>
+          ageSummary
+            .labels(clientId, topic)
+            .observe(age.toNanos.nanosToSeconds)
         }
+      } yield a
+    }
 
-        def count(name: String, topic: Topic) = {
-          callsCounter
-            .labels(clientId, topic, name)
-            .inc()
-        }
+    override def count(name: String, topic: Topic): F[Unit] = {
+      callsCounter
+        .labels(clientId, topic, name)
+        .inc()
+    }
 
-        def rebalance(name: String, topicPartition: TopicPartition) = {
-          rebalancesCounter
-            .labels(clientId, topicPartition.topic, name)
-            .inc()
-        }
+    override def rebalance(name: String, topicPartition: TopicPartition): F[Unit] = {
+      rebalancesCounter
+        .labels(clientId, topicPartition.topic, name)
+        .inc()
+    }
 
-        def topics(latency: FiniteDuration) = {
-          topicsLatency
-            .labels(clientId)
-            .observe(latency.toNanos.nanosToSeconds)
+    override def topics(latency: FiniteDuration): F[Unit] = {
+      topicsLatency
+        .labels(clientId)
+        .observe(latency.toNanos.nanosToSeconds)
+    }
+  }
+
+  def histograms[F[_]: Monad](
+    registry: CollectorRegistry[F],
+    prefix: Prefix = Prefix.Default
+  ): Resource[F, ClientId => ConsumerMetrics[F]] = {
+
+    val callsCounter = registry.counter(
+      name   = s"${prefix}_calls_total",
+      help   = "Number of topic calls",
+      labels = LabelNames("client", "topic", "type")
+    )
+
+    val resultCounter = registry.counter(
+      name   = s"${prefix}_results_total",
+      help   = "Topic call result: success or failure",
+      labels = LabelNames("client", "topic", "type", "result")
+    )
+
+    val latencyHistogram = registry.histogram(
+      name    = s"${prefix}_latency_seconds",
+      help    = "Topic call latency in seconds",
+      buckets = latencyBuckets,
+      labels  = LabelNames("client", "topic", "type")
+    )
+
+    val recordsHistogram = registry.histogram(
+      name    = s"${prefix}_poll_records_total",
+      help    = "Number of records per poll",
+      buckets = pollCountBuckets,
+      labels  = LabelNames("client", "topic")
+    )
+
+    val bytesHistogram = registry.histogram(
+      name    = s"${prefix}_poll_bytes",
+      help    = "Number of bytes per poll",
+      buckets = pollBytesBuckets,
+      labels  = LabelNames("client", "topic")
+    )
```

Comment:
currently we have other metric:
```scala
    val bytesSummary = registry.summary(
      name      = s"${prefix}_poll_bytes",
      help      = "Number of bytes per poll",
      quantiles = Quantiles.Empty,
      labels    = LabelNames("client", "topic")
    )
```

sees that these will collide and downstream users will have to notice it and amend Grafana panels

do we want to allow that? I am not sure, if this is the best way how to keep them

maybe we should explicitly add version infix like `_v2_` in metric new names?

(in all other `.summary` -> `.histogram` cases we added extra suffix - thus creating alternative set of metrics)